components/generator/input.vue

```diff
@@ -3,6 +3,13 @@
     <form
       v-on:submit.prevent="create"
       class="box">
+      <c-icon
```

Comment:
I think this should be feature toggled for a while for more directed testing before releasing to the wild